# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/openal/hotfix-gcc15.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- openal-soft-1.23.1/core/uhjfilter.h.vanilla	2025-04-28 22:01:00.644677438 +0300
+++ openal-soft-1.23.1/core/uhjfilter.h	2025-04-28 22:01:38.809603023 +0300
@@ -2,6 +2,7 @@
 #define CORE_UHJFILTER_H
 
 #include <array>
+#include <stdint.h>
 
 #include "almalloc.h"
 #include "alspan.h"
--- openal-soft-1.23.1/core/mixer/defs.h.vanilla	2025-04-28 22:00:35.183183295 +0300
+++ openal-soft-1.23.1/core/mixer/defs.h	2025-04-28 22:00:42.430652850 +0300
@@ -3,6 +3,7 @@
 
 #include <array>
 #include <stdlib.h>
+#include <stdint.h>
 
 #include "alspan.h"
 #include "core/bufferline.h"
--- openal-soft-1.23.1/alc/alu.h.vanilla	2025-04-28 22:00:02.422251764 +0300
+++ openal-soft-1.23.1/alc/alu.h	2025-04-28 22:00:48.359188988 +0300
@@ -2,6 +2,7 @@
 #define ALU_H
 
 #include <bitset>
+#include <stdint.h>
 
 #include "aloptional.h"
 
